src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import cn from 'classnames';
+
+import type { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  onDelete: (id: number) => Promise<number | void>;
+  hasLoader: boolean;
+}
+
+export const TodoItem: React.FC<Props> = React.memo((props) => {
+  const { todo, onDelete, hasLoader } = props;
+
+  return (
+    <div
+      data-cy="Todo"
+      className={cn('todo', { completed: todo.completed })}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
+          onChange={() => { }}
```

Comment:
It makes no sense to add empty handlers. It just clutters up the code

Reply:
I had a warning in the console without this handler: 
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/147749101/6e7b5604-a88e-4e7c-9ac7-1ed523fa0184)
so i decided add empty and in the next task I'll write normal handler for this anyway